# message file for object information
# trial id
int64 id
# size of object, 1: small, 2: big
int64 object_size
